stub/include/neurala/image/views/ImageView.h

```diff
@@ -110,79 +136,67 @@ class ImageView
 	/// @copydoc ImageMetadata::datatype() const
 	constexpr EDatatype datatype() const noexcept { return m_metadata.datatype(); }
 
-	/// @copydoc ImageMetadata::sizeBytes() const
-	constexpr size_type sizeBytes() const noexcept { return m_metadata.sizeBytes(); }
+	/// @brief Returns the type of view strategy used.
+	virtual std::string viewStrategy() const noexcept { return "identity"; }
```

Comment:
This return type is temporary (to be replaced with whatever equivalent for `EViewStrategy`).
I considered returning a vector of values, for various considered elements (ROI, mask, polygons). I suspect that is going too far; worst case, new values can be added to represent the various valid combinations.

Reply:
Update: as this class is getting moved to the backend, the return type can remain temporarily.